.pre-commit-config.yaml

```diff
@@ -7,10 +7,6 @@ repos:
       - id: mixed-line-ending
       - id: check-case-conflict
       - id: check-yaml
-  - repo: https://github.com/asottile/seed-isort-config
-    rev: v2.2.0
-    hooks:
-      - id: seed-isort-config
```

Comment:
@ravwojdyla removed seed-isort-config since it's deprecated with repo archived and apparently not needed since isort 5. We will no longer have a `known_third_party` config section populated. Does this make sense to you?